osquery/tables/CMakeLists.txt

```diff
@@ -82,7 +82,7 @@ endif()
 # TODO: These tables should be made Windows-compatible
 if(NOT WINDOWS)
   file(GLOB OSQUERY_POSIX_NETWORKING_TABLES "networking/posix/*.cpp")
-  file(GLOB OSQUERY_CROSS_FORENSIC_TABLES "forensic/*.cpp")
+  file(GLOB OSQUERY_POSIX_FORENSIC_TABLES "forensic/posix/*.cpp")
```

Comment:
If Windows linking is fixed (like this CMake suggests https://github.com/facebook/osquery/pull/3038/files#diff-c6785b971f7dd64f549327a71de0b3db) can this be CROSS again or are there other blockers?